fab/fabfile.py

```diff
@@ -1045,6 +1044,18 @@ def services_restart():
     _supervisor_command('start all')
 
 
+@roles(ROLES_DJANGO)
+@serial
+def _restart_webworkers():
+    _services_restart()
+
+
+@roles(set(ROLES_ALL_SERVICES) - set(ROLES_DJANGO))
+@parallel
+def _restart_services():
+    _services_restart()
```

Comment:
It's confusing to have both `_restart_services` and `_services_restart` without documentation to explain the difference (preferably the difference would be clear in the name). Suggested rename:
`_services_restart` -> `_restart_all_except_webworkers`